init_scripts/launch_osr.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# exit on error, and output executed commands to stdout
+set -ex
+
+source osr_paths.sh
+launch_dir=$OSR_CODE_DIR/ROS/osr_bringup/launch
+
+bash -c ". /home/$USER/osr_ws/devel/setup.sh"
```

Comment:
Wait, do you have this devel directory? I don't, after following the instructions in https://github.com/nasa-jpl/osr-rover-code/blob/foxy-devel/setup/rpi.md. Seems like it's never created, because we're not using `catkin_make`?

I had to use `source install/setup.bash` instead...

Reply:
```suggestion
bash -c ". /home/$USER/osr_ws/install/setup.sh"
```